src/library/simplified/extension/price-per-day.js

```diff
@@ -12,4 +12,7 @@ if (utag.isCarCO()) {
         pricePerDay = b['entity.checkout.car.pricePerDay'];
     } 
     b['pricePerDay'] = pricePerDay;
+}
+else if (utag.isCarUDP() && typeof b["entity.carDetails.carCategoryNameLocalized"] !== undefined) {
+    b['pricePerDay'] = b["entity.carDetails.carCategoryNameLocalized"];
```

Comment:
Is this correct? Do we want the category name to go into the price per day?

Reply:
tony, thanks for the comment. I missed it while creating PR. Incorporated that and updated PR